src/net.cpp

```diff
@@ -421,6 +421,9 @@ bool GetMyExternalIP(CNetAddr& ipRet)
 
 void ThreadGetMyExternalIP(void* parg)
 {
+    // Make this thread recognisable as the message handling thread
```

Comment:
This thread is not the message handling thread.
`// Make this thread recognisable as the external IP detection thread`